src/rayLength.js

```diff
@@ -6,15 +6,42 @@
     return Math.sqrt(a * a + b * b);
   }
 
-  function getRayLength (size, containerWidth, containerHeight, offsetPosX, offsetPosY) {
-    var distRight = Math.abs(containerWidth - offsetPosX);
-    var distBottom = Math.abs(containerHeight - offsetPosY);
+  function getDistanceToSides (distLeft, distTop, distRight, distBottom, angle) {
+    var s = Math.sin(angle * Math.PI / 180);
```

Comment:
nit: s and c as variable names is a bit unclear. can we use sin and cos? i don't think they're reserved